app/api/suggestions/updateState.ts

```diff
@@ -13,6 +13,8 @@ import {
   getLabeledValueStage,
 } from './pipelineStages';
 
+type SuggestionsAggregationResult = SuggestionValues & { _id: any; propertyName: string };
```

Comment:
What was the reason for taking these out?

Reply:
This was originally misdefined but was not evident until I tried some things (that finally removed).
The thing is: `SuggestionVales` is defined as the type of the input needed but the `getState` function, or what is needed to calculate the state.
The `_id` and `propertyName` were there because they were returned in the mongo query, but made no sense from the point of view of the submodule defining the interface. What makes sense is that we segregate the interfaces, and this query uses an extended version for it's purpose.